pkg/environment/environment.go

```diff
@@ -43,6 +44,7 @@ const (
 	DeafultCopyTemplateIMG      = "quay.io/kubevirt/tekton-task-copy-template:" + operands.TektonTasksVersion
 	DeafultCleanupVMIMG         = "quay.io/kubevirt/tekton-task-execute-in-vm:" + operands.TektonTasksVersion
 	GenerateSSHKeysIMG          = "quay.io/kubevirt/tekton-task-generate-ssh-keys:" + operands.TektonTasksVersion
+	DefaultVirtioIMG            = "quay.io/kubevirt/virtio-container-disk:20230119_27f97a427"
```

Comment:
I'd set that to tag `latest`. But please note that the virtio container disk is not updated very often. Although there are many tags of recent builds it was last changed in Apr 22.

See https://github.com/kubevirt/kubevirt/pull/7546

Reply:
I would rather set tag of kubevirt (in this case v0.58.0).

Reply:
How do we make sure it is updated and pointing to the matching kubevirt release?

Reply:
We can do the same github actions script as we are using in common templates

Reply:
You mean an action to update the version periodically in a PR so we have fixed versions in a release? I'd be okay with that.

Reply:
yes, that is exactly what I was thinking of.

Reply:
Please do that in a follow up PR.